lib/creator/yeoman/utils/entry.js

```diff
@@ -41,7 +41,7 @@ module.exports = (self, answer) => {
 			return forEachPromise(entryIdentifiers, (entryProp) => self.prompt([
 				InputValidate(
 					`${entryProp}`,
-					`What is the location of '${entryProp}'? [example: '../dist/app.js']`,
+					`What is the location of '${entryProp}'? [example: './app']`,
```

Comment:
It needs an 'entry file' location, entry files are **input** files, not **output**, so `build` makes no more sense than `dist`.

See the `app` and `vendors` examples here: https://webpack.js.org/concepts/entry-points/#separate-app-and-vendor-entries